build.sbt

```diff
@@ -53,6 +53,10 @@ lazy val ws = (project in file("acdc-ws")).
       // the transitive jackson dependencies from play framework on has security vulnerabilities
       "com.fasterxml.jackson.core" % "jackson-databind" % "2.15.3",
       "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.15.3"
-    )
+    ),
+    assemblyMergeStrategy in assembly := {
+      case PathList("META-INF", _*) => MergeStrategy.discard
+      case _                        => MergeStrategy.first
+    }
```

Comment:
fixes assembly errors i encountered building with a clean .ivy cache

Reply:
you sure? CircleCI with clean cache doesn't have any issues with assembly